### PR TITLE
IMP update to libbeat 5.6

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: _meta/beat.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+################### Mysqlbeat Configuration Example #########################
+
+############################# Mysqlbeat ######################################
+
+mysqlbeat:
+  # Defines how often an event is sent to the output
+  period: 1s
```
Comment (mdelaney) [inline _meta/beat.yml:7]: 1s is probably a little excessive. How about 10s?

File: _meta/kibana/5.x/index-pattern/mysqlbeat.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  "fieldFormatMap": "{\"@timestamp\": {\"id\": \"date\"}}", 
+  "timeFieldName": "@timestamp", 
+  "title": "mysqlbeat-*"
+}
```
Comment (mdelaney) [inline _meta/kibana/5.x/index-pattern/mysqlbeat.json:6]: add newline at end of file (this applies to a few other files in this PR)

File: tests/system/config/mysqlbeat.yml.j2 [hunk 3]
```diff
@@ -14,12 +14,12 @@ output:
     enabled: true
 
     # Path to the directory where to save the generated files. The option is mandatory.
-    path: {{ output_file_path|default(beat.working_dir + "/output") }}
+    path: {{ '{{' }} output_file_path|default(beat.working_dir + "/output") {{ '}}' }}
```
Comment (mdelaney) [inline tests/system/config/mysqlbeat.yml.j2:17]: Why are we escaping `{{` here? Isn't the purpose to actually evaluate and insert the value of `output_file_path|default(beat.working_dir + "/output")`? (comment applies to line 22 as well)

**Review by mdelaney — CHANGES_REQUESTED** [review]
The templates also do not specify mappings for fields that this beat is adding to each document. The expected fields should probably be added.

**Comment by mdelaney** [comment]
ahh, I see that https://github.com/Tubular/mysqlbeat/pull/2 adds the field mapping.

Lets get this PR comments addressed and merged then rebase the other PR and we can look at it then.

File: tests/system/config/mysqlbeat.yml.j2 (hunk 3 above)
Comment (ghost) [reply tests/system/config/mysqlbeat.yml.j2:17]: @mdelaney it's autogenerated by libbeat 
link to similar at burrowbeat https://github.com/Tubular/burrowbeat/blob/develop/tests/system/config/burrowbeat.yml.j2#L30

File: _meta/kibana/5.x/index-pattern/mysqlbeat.json (hunk 2 above)
Comment (ghost) [reply _meta/kibana/5.x/index-pattern/mysqlbeat.json:6]: will be addressed in new PR

File: _meta/beat.yml (hunk 1 above)
Comment (ghost) [reply _meta/beat.yml:7]: Yep in next PR, changed to 1m interval

**Review by mdelaney — APPROVED** [review]
as long as comments are addressed in next PR

**Pull request merged**